packages/@aws-cdk/aws-codebuild/test/integ.project-vpc.ts

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+import ec2 = require('@aws-cdk/aws-ec2');
+import cdk = require('@aws-cdk/cdk');
+import { Project } from '../lib';
+
+const app = new cdk.App();
+
+const stack = new cdk.Stack(app, 'aws-cdk-codebuild-project-vpc');
+const vpc = new ec2.VpcNetwork(stack, 'MyVPC');
+const securityGroup = new ec2.SecurityGroup(stack, 'SecurityGroup1', {
+    groupName: 'Bob',
+    vpc: vpc,
+    allowAllOutbound: true,
+    description: 'Example',
+});
+new Project(stack, 'MyProject', {
+    vpcConfig: {
+        vpc: vpc,
+        securityGroupIDs: [securityGroup.securityGroupId],
+        subnetsIDs: [vpc.privateSubnets[0].subnetId]
+    }
+});
+
+app.run();
```

Comment:
You might have forgot to commit `integ.project-vpc.expected.json` :)

Reply:
Fixed